tests/openwisp2/settings.py

```diff
@@ -278,7 +278,7 @@
 
 # OPENWISP_RADIUS_PASSWORD_RESET_URLS = {
 #     # fallback in case the specific org page is not defined
-#     'default': 'https://example.com/{{organization}/password/reset/confirm/{uid}/{token},
+#     '__all__': 'https://example.com/{{organization}/password/reset/confirm/{uid}/{token},
 #     # use the uuid because the slug can change
 #     # 'dabbd57a-11ca-4277-8dbb-ad21057b5ecd': 'https://org.com/{organization}/password/reset/confirm/{uid}/{token}',
 # }
```

Comment:
@nemesisdesign @codesankalp why is this commented code present here? Is it present for instructions? If so, can we move the instructions to the part where the `__all__` key is configured?

Reply:
I think it is for [instructions](https://github.com/openwisp/openwisp-radius/commit/fb663c69a2891aadea416d63e491ba44a53fe040#diff-cf01e5d691c05467162c955fc1688cc98afdb2ad6c6640caacb2569b0735a7c7)

I will move this block above `__all__` block.